src/main/java/model/Line.java

```diff
@@ -0,0 +1,30 @@
+package model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class Line {
+    private List<Boolean> points = new ArrayList<>();
```

Comment:
Boolean을 넣어주는 것도 좋지만 Boolean말고 다른 객체를 만들어 List를 만들어 보면 어떨까요??

```suggestion
    private List<CorssBar> crossBars = new ArrayList<>();
```

이런식으로 만들어 보면 어떨까요? :)